fdbserver/DataDistribution.actor.cpp

```diff
@@ -474,6 +474,7 @@ ACTOR Future<Reference<InitialDataDistribution>> getInitialDataDistribution( Dat
 			try {
 				tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
 				wait(checkMoveKeysLockReadOnly(&tr, moveKeysLock));
+				state Standalone<RangeResultRef> UIDtoTagMap = wait(tr.getRange(serverTagKeys, GetRangeLimits()));
```

Comment:
If the `getRange` cannot finish because it has too many results, will the error be explicit enough that we can catch the problem quickly? 
I'm ok with either way as long as we can get a more explicit result when `getRange` fails due to too_many keys.

Reply:
The only reason to have TOO_MANY is just to avoid running the process out of memory if something crazy happens, and have a more intelligent error if it was doing something unexpected. I generally prefer this method, but do not have strong opinions.